website/docs/components/form/primitives/partials/guidelines/overview.md

```diff
@@ -7,6 +7,8 @@ Form Primitives are used to compose form fields.
 - `Form::Legend` is the legend associated to the fieldset
 - `Form::Field` is the generic container for control, label, helper text and error messaging
 - `Form::Fieldset` is the generic container to group multiple fields with label, helper text, and error messaging
+- `Form::CharacterCount` is optionally used to display the number of characters entered in a field, the maximum number of characters allowed, or a custom message communicating the relationship between the count and the maximum length.
+    - `CharacterCount` is only used in fields that accept text values (TextInput, Textarea, MaskedInput, etc).
```

Comment:
Removed "etc" since it's not available in any other fields at this time. 

```suggestion
    - `CharacterCount` is only used in fields that accept text values (TextInput, Textarea, and MaskedInput).
```